packages/vanilla/src/components/checkbox/checkbox.stories.js

```diff
@@ -0,0 +1,165 @@
+      description: 'Represents a caption for the content of its parent `<fieldset>`.',
+      control: { type: 'text' }
+    },
+    inputDescription: {
+      name: 'Input Description',
+      description: 'Instructions or supplementary information regarding the inputs in the `<fieldset>`. Placed below the `<legend>` element.',
+      control: { type: 'text' }
+    },
+    errorMessage: {
+      name: 'Error Message',
+      description: 'A message in the checkbox description that a problem has occurred.',
+      control: { type: 'text' }
+    },
+    disabled: {
+      name: 'Disabled',
+      description: 'Disable checkbox. **In order to see a "checked" disabled checkbox, toggle the Checked story control to `true`**',
+      control: { type: 'boolean' }
+    }
+  },
+  decorators: [
+    (Story, context) => `
+      <fieldset class="cbp-fieldset ${context.args.displayInline ? 'cbp-fieldset--inline' : ''}">
+        <legend class="cbp-legend">${context.args.legend}</legend>
+        <p class="cbp-input__description">${context.args.inputDescription}</p>
+        <p class="cbp-input__description cbp-input__description--error" hidden><i class="fas fa-exclamation-triangle"></i>${context.args.errorMessage}</p>
+        ${Story().outerHTML || Story()}
+      </fieldset>
+    `
+  ]
+};
+
+const Template = ({ formControlName, checkboxObj: { checkbox1, checkbox2, checkbox3 }, disabled, checked }) => (
+  `
+    <div class="cbp-checkbox-item cbp-margin-bottom-5x">
+      <input type="checkbox" name=${formControlName} class="cbp-input__checkbox" id=${checkbox1.id} value=${checkbox1.value} ${disabled ? 'disabled' : ''} ${checked ? 'checked' : ''}>
+      <label for=${checkbox1.id}>${checkbox1.label}</label>
+    </div>
+    <div class="cbp-checkbox-item cbp-margin-bottom-5x">
+      <input type="checkbox" name=${formControlName} class="cbp-input__checkbox" id=${checkbox2.id} value=${checkbox2.value} ${disabled ? 'disabled' : ''}>
+      <label for=${checkbox2.id}>${checkbox2.label}</label>
+    </div>
+    <div class="cbp-checkbox-item">
+      <input type="checkbox" name=${formControlName} class="cbp-input__checkbox" id=${checkbox3.id} value=${checkbox3.value} ${disabled ? 'disabled' : ''}>
+      <label for=${checkbox3.id}>${checkbox3.label}</label>
+    </div>
+  `
+)
+
+const IndeterminateCheckboxTemplate = ({ indeterminateCheckboxObj, nestedCheckboxObj: { name: nestCheckboxName, checkbox1, checkbox2, checkbox3 }, disabled }) => {
+  return `
+    <div class="cbp-checkbox-item cbp-margin-bottom-5x">
+      <input type="checkbox" name=${indeterminateCheckboxObj.name} class="cbp-input__checkbox" id=${indeterminateCheckboxObj.id} value=${indeterminateCheckboxObj.value} data-checkbox="indeterminate" ${disabled ? 'disabled' : ''}>
+      <label for=${indeterminateCheckboxObj.id}>${indeterminateCheckboxObj.label}</label>
+    </div>
+    <div class="cbp-checkbox--nested">
+      <div class="cbp-checkbox-item cbp-margin-bottom-5x">
```

Comment:
I would probably use flexbox and gap for the list spacing. It's more obvious for radios where you allow switching between vertical and horizontal.